src/main/java/net/sf/jabref/specialfields/SpecialFieldUpdateListener.java

```diff
@@ -15,30 +15,28 @@
 */
 package net.sf.jabref.specialfields;
 
-import java.beans.PropertyChangeEvent;
-import java.beans.PropertyVetoException;
-import java.beans.VetoableChangeListener;
-
 import javax.swing.SwingUtilities;
 
 import net.sf.jabref.JabRefGUI;
+import net.sf.jabref.event.ChangeSpecialFieldEvent;
 import net.sf.jabref.gui.undo.NamedCompound;
 import net.sf.jabref.model.entry.BibEntry;
 
+import com.google.common.eventbus.Subscribe;
+
 /**
  * Listener triggering
  *  * an update of keywords if special field has been updated
  *  * an update of special fields if keywords have been updated
  */
-public class SpecialFieldUpdateListener implements VetoableChangeListener {
+public class SpecialFieldUpdateListener {
 
     private static SpecialFieldUpdateListener INSTANCE;
 
-
-    @Override
-    public void vetoableChange(PropertyChangeEvent e) throws PropertyVetoException {
-        final BibEntry entry = (BibEntry) e.getSource();
-        final String fieldName = e.getPropertyName();
+    @Subscribe
+    public void listen(ChangeSpecialFieldEvent changeSpecialFieldEvent) {
```

Comment:
It should listen for `ChangeFieldEvent`, not for this special class. Please remove the class `ChangeSpecialFieldEvent`.